### DISCUSSION TITLE
npx create-expo-app@latest reads wrong node version

### DISCUSSION THREAD

**Discussion by SauravKumarJhaNITW** [expo/expo#31149] [Archived: Questions]
I have node version 20.17.0 installed on my windows machine. 
command `node -v` prints the same.
but when I run below expo command, 
`npx create-expo-app@latest`

it gives the following warning:

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'react-native@0.74.5',
npm WARN EBADENGINE   required: { node: '>=18' },
npm WARN EBADENGINE   current: { node: 'v17.7.2', npm: '10.8.2' }
npm WARN EBADENGINE }
```

I have no idea from where it is detecting node version as 17.7.2 ?
can someone help on this please?
I have tried uninstalling/reinstalling node expo-cli, clearing npm-cache and what not but this warning keeps coming.

**Comment by AtriAhmed** [comment]
did you find a solution for this ?

**Reply by SauravKumarJhaNITW** [reply]
no, I could not find any solution, but I switched to wsl (windows subsystem for linux) and it worked.
also, changing to other machine worked. I have no idea of the core issue.
